joss/paper.md

```diff
@@ -29,14 +29,14 @@ bibliography: paper.bib
 ---
 
 # Summary
-Across scientific fields, the Bayesian framework is used to account for uncertainties in inference [@ellison2004bayesian; @lancaster2004introduction; @von2011bayesian]. However, for models with more than a few parameters, exact inference is intractable. A frequently used strategy is to approximately sample the posterior distribution on models' parameters with a Markov chain Monte Carlo (MCMC). Yet conventional MCMC methods relying on local updates can take a prohibitive time to converge when posterior distributions have complex geometries (see e.g., [@rubinstein_simulation_2017] for a textbook).
+Across scientific fields, the Bayesian framework is used to account for uncertainties in inference [@ellison2004bayesian; @lancaster2004introduction; @von2011bayesian]. However, for models with more than a few parameters, exact inference is intractable. A frequently used strategy is to approximately sample the posterior distribution on a model's parameters with a Markov chain Monte Carlo (MCMC) method. Yet conventional MCMC methods relying on local updates can take a prohibitive time to converge when posterior distributions have complex geometries (see e.g., @rubinstein_simulation_2017).
```

Comment:
At the moment there is some switching between plural and singular forms - it should either be
> sample the posterior distributions on models' parameters
or
> sample the posterior distribution on a model's parameters
I've suggested latter (singular) option as think it reads better.

'A Markov chain Monte Carlo' doesn't read right - I think either should be no determiner or use as qualifier for 'method' as suggested.

The `rubinstein_simulation_2017` citation shouldn't be in parenthesis as its referred to directly, and I think 'for a textbook' doesn't scan right and isn't necessary.